modules/auxiliary/scanner/http/log4shell_scanner.rb

```diff
@@ -13,10 +13,16 @@ def initialize
     super(
       'Name' => 'Log4Shell HTTP Scanner',
       'Description' => %q{
+        Apache Log4j2 <=2.14.1 JNDI features used in configuration, log messages, and parameters do not protect against
```

Comment:
They're saying now that even 2.15 is vulnerable. Can you update this to reflect that?